lib/perl/Genome/Model/ClinSeq/OriginalScripts/adhoc/luc/processLuc17Data.pl

```diff
@@ -744,7 +744,7 @@
     print YELLOW, "\n\tTophat summary dir already exists - skipping this step", RESET;    
   }else{
     mkdir($tophat_summary_dir);
-    my $summary_cmd = "/gscmnt/sata206/techd/git/genome/lib/perl/Genome/Model/ClinSeq/original-scripts/qc/tophatAlignmentSummary.pl  --reference_fasta_file='/gscmnt/sata420/info/model_data/2857786885/build102671028/all_sequences.fa'  --reference_annotations_dir='/gscmnt/sata132/techd/mgriffit/reference_annotations/hg19/'  --working_dir=$tophat_summary_dir  --tophat_alignment_dir='$rnaseq_alignments_dir'  --verbose=1";
+    my $summary_cmd = "/gscmnt/sata206/techd/git/genome/lib/perl/Genome/Model/ClinSeq/original-scripts/qc/tophatAlignmentSummary.pl  --reference_fasta_file='/gscmnt/gc4096/info/model_data/2857786885/build102671028/all_sequences.fa'  --reference_annotations_dir='/gscmnt/sata132/techd/mgriffit/reference_annotations/hg19/'  --working_dir=$tophat_summary_dir  --tophat_alignment_dir='$rnaseq_alignments_dir'  --verbose=1";
```

Comment:
There's no copy of Genome in either location and there's no longer an `original-scripts` directory in the repository, anyway.  (It was renamed to `OriginalScripts`.)

This isn't really the right way to go about referring to other scripts in the directory, but since this has long not worked, I didn't worry about it.